docs/source/_json/types.resp

```diff
@@ -18,6 +18,11 @@ Content-Type: application/json
     "title": "DXTestDocument"
   }, 
   {
+    "@id": "http://localhost:55001/plone/@types/DXTestFolder", 
```

Comment:
@jaroel can you elaborate why you need the DXTestFolder type here? I know we added this for AT. Though, I somehow dislike test profiles because they obfuscate the test setup (you need to look into many different external files to understand your test set up). A test type can be easily set up within Python code and this improves the readability of the test. 

Therefore I would like to get rid of the ATTestFolder in the profile instead of adding the DX equivalent.